bootstrap/config/config.go

```diff
@@ -25,15 +25,21 @@ import (
 
 	"github.com/edgexfoundry/go-mod-configuration/configuration"
 	configTypes "github.com/edgexfoundry/go-mod-configuration/pkg/types"
+
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 )
 
-const (
-	WritableKey = "/Writable"
-)
+const writableKey = "/Writable"
+
+// UpdatedStream defines the stream type that is notified by ListenForChanges when a configuration update is received.
+type UpdatedStream chan<- struct{}
```

Comment:
Need to send copy of previous Writable before updated, so I think needs to be `interface{}` rather than `struct{}`.